packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -209,7 +212,12 @@ function InserterBlockList( {
 
 			{ ! hasChildItems &&
 				map( categories, ( category ) => {
-					const categoryItems = itemsPerCategory[ category.slug ];
+					const categoryItems =
+						itemsPerCategory[
+							category === defaultCategory
+								? undefined
+								: category.slug
+						];
```

Comment:
This works, but it's a bit strange, because it's assuming there's a string key `undefined` in `itemsPerCategory`, i.e.

```
itemsPerCategory = {
   // ...
   undefined: [ /* ... */ ],
};
```